### DISCUSSION TITLE
Crest, Shape and Impulse factor

### DISCUSSION THREAD

**Discussion by vutle** [blue-yonder/tsfresh#865] [Ideas]
Could we add the crest factor inside the feature set please.

![image](https://user-images.githubusercontent.com/6241928/121771083-7efb1980-cbb0-11eb-8044-51ef5cc2b851.png)

**Comment by nils-braun** [comment]
Sure! Thanks for the suggestion. 
Would you like to open a PR for these features? We welcome any contribution!
Have a look into the docu [here](https://tsfresh.readthedocs.io/en/latest/text/how_to_add_custom_feature.html) and [here]https://tsfresh.readthedocs.io/en/latest/text/how_to_contribute.html] if this is your first-time contribution to tsfresh.

Would love to see this!
